pylint/checkers/refactoring/recommendation_checker.py

```diff
@@ -121,6 +121,12 @@ def _check_use_maxsplit_arg(self, node: nodes.Call) -> None:
             and isinstance(utils.safe_infer(node.func), astroid.BoundMethod)
         ):
             return
+        inferred_expr = utils.safe_infer(node.func.expr)
+        if not isinstance(inferred_expr, nodes.Const) and (
+            isinstance(inferred_expr, astroid.Instance)
```

Comment:
Can't the first line be removed? The `and` makes the first check redundant.

Reply:
updated :) thanks again